src/experimental/ProbabilisticGraphicalModels/bayesian_network.jl

```diff
@@ -1,3 +1,7 @@
+using MetaGraphsNext
+using Graphs
+using Distributions
```

Comment:
```suggestion

```
these should not be strictly necessary. It is generally a good idea to import/using all the packages at the top level.